released/usecases/services_apps/sap_build_code/step2/main.tf

```diff
@@ -0,0 +1,269 @@
+# ------------------------------------------------------------------------------------------------------
+# Create the Cloud Foundry space
+# ------------------------------------------------------------------------------------------------------
+resource "cloudfoundry_space" "dev" {
+  name = "dev"
+  org  = var.cf_org_id
+}
+
+# ------------------------------------------------------------------------------------------------------
+# SETUP ALL SERVICES FOR CF USAGE
+# ------------------------------------------------------------------------------------------------------
+# ------------------------------------------------------------------------------------------------------
+# Setup sdm
+# ------------------------------------------------------------------------------------------------------
+# Entitle 
+resource "btp_subaccount_entitlement" "sdm" {
```

Comment:
Wouldn't it make sense to add the entitlement in step 1 -> the the propagation to the CF Marketplace took place before creating the service instance?

Reply:
Revised: To keep things self-contained, we leave the stuff in the CF step